src/sr/discord_bot/commands/team.py

```diff
@@ -255,3 +256,37 @@ async def export_team(
         output = output + await _export_team(tla, only_teams, guild, interaction)
     output = output + "\n```"
     await interaction.followup.send(content=output, ephemeral=True)
+
+
+def _repair_permissions_status_msg(cur_team: int, num_teams: int) -> str:
+    return f"Repairing permissions... {cur_team}/{num_teams} teams processed"
+
+
+@group.command(  # type:ignore[arg-type]
+    name='repair-permissions',
+    description='Reset channel and team permissions',
+)
+async def repair_permissions(interaction: discord.interactions.Interaction["BotClient"]) -> None:
+    guild: discord.Guild | None = interaction.guild
+    if guild is None:
+        raise app_commands.NoPrivateMessage()
+
+    team_roles: list[discord.Role] = [t for t in guild.roles if t.name.startswith(ROLE_PREFIX)]
```

Comment:
Hopefully shouldn't need the explicit type annotation here since it should come from the comprehension. Also mildly suggest a longer variable name:
```suggestion
    team_roles = [role for role in guild.roles if role.name.startswith(ROLE_PREFIX)]
```